pkg/prometheus/operator.go

```diff
@@ -73,6 +73,45 @@ type Operator struct {
 	config                 Config
 }
 
+type Labels struct {
+	LabelsString string
+	LabelsMap    map[string]string
+}
+
+// Implement the flag.Value interface
+func (labels *Labels) String() string {
+	return fmt.Sprintf(labels.LabelsString)
```

Comment:
`return labels.LabelsString` should be equivalent, and since we're not using `.LabelsString` outside of this file, we can leave it unexported.

Reply:
yes, that's a mistake from previous code